pkg/controller/resource/daemonset.go

```diff
@@ -27,45 +27,42 @@ import (
 	chaosTypes "github.com/litmuschaos/chaos-operator/pkg/controller/types"
 )
 
-// CheckDaemonSetAnnotation will check the annotation of DaemonSet
+// CheckDaemonSetAnnotation check the annotation of the DaemonSet
 func CheckDaemonSetAnnotation(clientset kubernetes.Interface, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, error) {
 	targetAppList, err := getDaemonSetLists(clientset, engine)
 	if err != nil {
 		return engine, err
 	}
-	engine, chaosEnabledDaemonSet, err := checkForChaosEnabledDaemonSet(targetAppList, engine)
-	if err != nil {
-		return engine, err
-	}
+	engine, chaosEnabledDaemonSet := checkForChaosEnabledDaemonSet(targetAppList, engine)
 	if chaosEnabledDaemonSet == 0 {
 		return engine, errors.New("no chaos-candidate found")
 	}
 	return engine, nil
 }
 
-// getDaemonSetLists will list the daemonSets which having the chaos label
+// getDaemonSetLists returns a list of daemonsets that are found in the app namespace with specified label
 func getDaemonSetLists(clientset kubernetes.Interface, engine *chaosTypes.EngineInfo) (*appsV1.DaemonSetList, error) {
-	targetAppList, err := clientset.AppsV1().DaemonSets(engine.AppInfo.Namespace).List(metaV1.ListOptions{
+	targetAppList, err := clientset.AppsV1().DaemonSets(engine.Instance.Spec.Appinfo.Appns).List(metaV1.ListOptions{
 		LabelSelector: engine.Instance.Spec.Appinfo.Applabel,
-		FieldSelector: ""})
+	})
 	if err != nil {
 		return nil, fmt.Errorf("error while listing daemonSets with matching labels %s", engine.Instance.Spec.Appinfo.Applabel)
 	}
 	if len(targetAppList.Items) == 0 {
-		return nil, fmt.Errorf("no daemonSets apps with matching labels %s", engine.Instance.Spec.Appinfo.Applabel)
+		return nil, fmt.Errorf("no daemonSets found with matching labels %s", engine.Instance.Spec.Appinfo.Applabel)
 	}
 	return targetAppList, err
 }
 
-// checkForChaosEnabledDaemonSet will check and count the total chaos enabled application
-func checkForChaosEnabledDaemonSet(targetAppList *appsV1.DaemonSetList, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, int, error) {
+// checkForChaosEnabledDaemonSet check and count the total chaos enabled application
+func checkForChaosEnabledDaemonSet(targetAppList *appsV1.DaemonSetList, engine *chaosTypes.EngineInfo) (*chaosTypes.EngineInfo, int) {
 	chaosEnabledDaemonSet := 0
 	for _, daemonSet := range targetAppList.Items {
 		annotationValue := daemonSet.ObjectMeta.GetAnnotations()[ChaosAnnotationKey]
 		if IsChaosEnabled(annotationValue) {
-			chaosTypes.Log.Info("chaos candidate of", "kind:", engine.AppInfo.Kind, "appName: ", daemonSet.ObjectMeta.Name, "appUUID: ", daemonSet.ObjectMeta.UID)
+			chaosTypes.Log.Info("chaos candidate of", "kind:", engine.Instance.Spec.Appinfo.AppKind, "appName: ", daemonSet.ObjectMeta.Name, "appUUID: ", daemonSet.ObjectMeta.UID)
```

Comment:
```suggestion
			chaosTypes.Log.Info("chaos candidate for rollout", "kind", engine.Instance.Spec.Appinfo.AppKind, "appName", daemonSet.ObjectMeta.Name, "appUUID", daemonSet.ObjectMeta.UID)
```

Reply:
Btw, I don't thinking printing `UUID` is helpful anytime, will debugging. Lets keep chaosEngine details to track it down.